### PR TITLE
Add ability to show population number as well for Bru

### PR DESCRIPTION
**Pull request by Miroito**

### CODE REVIEW

File: src/modules/commands/discord/population.ts [hunk 1]
```diff
@@ -20,10 +21,12 @@ type Result = {
...
   keyword: 'population',
   description: 'display the current population',
-  help: 'Usage: `{prefix}population` - displays the current population',
+  help: 'Usage:\n `{prefix}population` - displays the current population\n`{prefix}population numbers` - Diplays the current population with numbers',
   alias: ['pop'],
   callback: async ({ args, reply, raw }) => {
-    if (args.length > 0) return
+    validateArgumentRange(args.length, 0, 1)
+
+    const numbersBool = args.length === 1 && args[0] === 'numbers'
```
Comment (vostrnad) [inline src/modules/commands/discord/population.ts:29]: You need to reply with an error when the argument is something else than "numbers".

File: src/modules/commands/discord/population.ts [hunk 2]
```diff
@@ -20,10 +21,12 @@ type Result = {
 export default new Command<discord.Message>({
   keyword: 'population',
   description: 'display the current population',
-  help: 'Usage: `{prefix}population` - displays the current population',
+  help: 'Usage:\n `{prefix}population` - displays the current population\n`{prefix}population numbers` - Diplays the current population with numbers',
```
Comment (vostrnad) [inline src/modules/commands/discord/population.ts:24]: 1. Please remove space after newline.
2. "displays" needs to be lowercase.

File: src/modules/commands/discord/population.ts [hunk 3]
```diff
@@ -50,19 +53,27 @@ export default new Command<discord.Message>({
...
+        numbersBool
+          ? population.vs
+          : Math.round(divide(population.vs, totalPop) * 100).toString() + '%'
+      }`,
+      `${getEmoji(raw.channel, 'faction_logo_ns')?.toString() || ' NS'} ${
+        numbersBool
+          ? population.ns
+          : Math.round(divide(population.ns, totalPop) * 100).toString() + '%'
+      }`,
+    ].join('      ')
```
Comment (vostrnad) [inline src/modules/commands/discord/population.ts:76]: Why increase the number of spaces?

**Review by vostrnad — CHANGES_REQUESTED** [review]

File: src/modules/commands/discord/population.ts (hunk 3 above)
Comment (Miroito) [reply src/modules/commands/discord/population.ts:76]: It makes it a little nicer

Comment (Miroito) [reply src/modules/commands/discord/population.ts:76]: Like hostis suggested

**Review by vostrnad — APPROVED** [review]

**Pull request merged**